cli/Application/Command/Make/Langtemplates.php

```diff
@@ -73,7 +73,7 @@ public function execute()
 		$this->getApplication()->outputTitle('Make Language templates');
 
 		ExtensionHelper::addDomainPath('Core', JPATH_ROOT . '/src');
-		ExtensionHelper::addDomainPath('CoreJS', JPATH_ROOT . '/www/media');
+		ExtensionHelper::addDomainPath('CoreJS', JPATH_ROOT . '/www/media/js');
 		ExtensionHelper::addDomainPath('Template', JPATH_ROOT . '/cache/twig');
```

Comment:
TBH the path isn't very pretty. Since it is a temporary thing it should go into a `temp` folder but since we don't have that I (mis)used the `cache` folder.
Looking at the code I think the script creates a `JTracker` sub directory to generate the language files, so there really shouldn't be any issues if you use the same base path for "real" caching stuff...